src/blocks/recommendations/README.md

```diff
@@ -1,44 +1,63 @@
 
-The Recommendations Block is designed to showcase links to content on your site as provided by the [Parse.ly `/related` API endpoint](https://www.parse.ly/help/api/recommendations#get-related).
+## Requirements
 
-You can add it in `Posts`, `Pages`, or nearly any other custom post type. The Block can also be used in Full Site Editing (FSE) mode or as a [Block-based Widget](https://wordpress.org/support/article/block-based-widgets-editor/).
+The Block is available only in WordPress 5.9 and higher.
 
-## How to Use
+## How to use
 
-- Inside the Block Editor, add the `Parse.ly Recommendations` block via the [standard block controls](https://wordpress.org/support/article/adding-a-new-block/).
-- Use the Block and Inspector Controls to configure the [block attributes](#block-attributes).
+Inside the Block Editor, add the `Parse.ly Recommendations` Block via the [standard block controls](https://wordpress.org/support/article/adding-a-new-block/) and configure its [settings](#settings) using the sidebar.
 
-## Block Attributes
+## Customizing the appearance
 
-### `boost`
+The Block's appearance is minimal and inherits the active theme's styling. At this moment, the Block doesn't offer any styling preferences in its settings and its appearance is meant to be manipulated using CSS.
 
-Passed to the API endpoint to determine how to rank prospective results.
+## Settings
 
-Default: `views`
+### Title
 
-### `imagestyle`
+Change the Block's title by updating this setting. You can also make the title section completely vanish by specifying an empty value.
 
-Default: `original`
+### Maximum Results
 
-### `limit`
+Specify the maximum number of recommendations to show.
 
-Default: `3`
+> Note: In certain cases, the number of results might be lower than this setting (especially if it is set to a large number).
 
-### `showimages`
+### Show Images
 
-Default: `true`
+Toggle this setting to enable or disable images in the results. 
 
-### `sort`
+### Image Style
 
-Default: `score`
+Use this setting to specify whether the Block should display the post's original image or the respective thumbnail provided by Parse.ly. Selecting the thumbnail option could be more performant but is not ideal for displaying large images.
 
-### `tag`
+> Note: If you select the `Parse.ly thumbnail` setting, the image will appear big and distorted. You should use CSS to tailor it to your needs.
 
-Default: none
+### Sort Recommendations
 
-### `title`
+Sort the results by score or published date. 
 
-Default: `Related Content`
+### Boost
+
+Sub-sort the results by a variety of [available metrics](https://www.parse.ly/help/api/available-metrics).
+
+### Additional CSS class(es) (Advanced section)
+
+For styling purposes, you can specify one or more CSS classes that will be assigned to the block. 
+
+## FAQ
+
+### I can't find the Block.
+
+Ensure that you're running WordPress version 5.9.0 or higher, as the Block is unavailable in lower WordPress versions.
```

Comment:
Make sure they don't have the Proxy disabled?

Reply:
Good catch. I had thought about it but then forgot.

Reply:
When the proxy is disabled, the block itself will display: `The REST route is unavailable. To use it, wp_parsely_enable_related_api_proxy should be true.`.

I think that this should be enough, but let me know if you would also like to have it in the docs.

Reply:
I agree it's enough, no need to include it in the docs.